openvm/src/powdr_extension/plonk/chip.rs

```diff
@@ -48,14 +48,14 @@ impl<P: IntoOpenVm> PlonkChip<P> {
     #[allow(dead_code)]
     pub(crate) fn new(
         precompile: PowdrPrecompile<P>,
+        original_airs: BTreeMap<usize, SymbolicMachine<P>>,
```

Comment:
We do not store the original airs in the precompile anymore, since they are common to all precompiles. We pass them here instead.